public-ui/src/app/app-routing.tsx

```diff
@@ -1,55 +1,129 @@
-import {Component as AComponent} from '@angular/core';
+import { Component as AComponent } from '@angular/core';
+import { AppRoute, AppRouter } from 'app/components/app-router';
+import { BaseReactWrapper } from 'app/data-browser/base-react/base-react.wrapper';
+import { EhrViewReactComponent } from 'app/data-browser/views/ehr-view/ehr-view-react.component';
+import { FitbitReactComponent } from 'app/data-browser/views/fitbit-view/fitbit-view-react.component';
+import { IntroVidReactComponent } from 'app/data-browser/views/intro-vids/intro-vids-react.component';
+import { PMReactComponent } from 'app/data-browser/views/pm/pm-react.component';
+import { dBHomeComponent } from 'app/data-browser/views/quick-search/home-view-react.component';
 import * as React from 'react';
-
-import {AppRoute, AppRouter} from 'app/components/app-router';
-import {BaseReactWrapper} from 'app/data-browser/base-react/base-react.wrapper';
-import {FitbitReactComponent} from 'app/data-browser/views/fitbit-view/fitbit-view-react.component';
-import {IntroVidReactComponent} from 'app/data-browser/views/intro-vids/intro-vids-react.component';
-import {PMReactComponent} from 'app/data-browser/views/pm/pm-react.component';
+import { SurveyViewReactComponent } from './data-browser/views/survey-view/survey-react-view.component';
 
 export const AppRoutingComponent: React.FunctionComponent = () => {
+
   return <AppRouter>
+    <AppRoute
+      path='/'
+      component={() => dBHomeComponent(
+        {
+          routeData: {
+            title: 'Data Browser',
+            breadcrumb: { value: '' }
+          }
+        })}
+    />
+    <AppRoute
+      path='/:id'
+      component={() => dBHomeComponent(
+        {
+          routeData: {
+            title: 'Data Browser',
+            breadcrumb: { value: '' }
+          }
+        })}
+    />
```

Comment:
I think this route is causing the issue with physical measurements and fitbit views. AFAICT we should be able to remove it, doesn't seem to be used anywhere unless I'm missing something